web/app/assets/javascripts/applications.js

```diff
@@ -52,4 +52,17 @@
     }
 
   window.ApplicationsApp = ApplicationsApp;
+
+  window.manualGithubRepo = function (select) {
+    if (select.value == '') {
+      input = document.createElement('input');
```

Comment:
If you don't prefix a variable declaration with `var`, `const`, or `let`, JS implicitly hoists it to the global scope, which is not what you want. Same with `parent` below.